cli.js

```diff
@@ -46,6 +47,7 @@ const cli = meow(`
 	  ${chalk.yellow('--stdin')}           Read input from stdin rather than from arguments.
 	  ${chalk.yellow('--no-newline, -n')}  Don't emit a newline (\`\\n\`) after the input.
 	  ${chalk.yellow('--demo')}            Demo of all Chalk styles.
+	  ${chalk.yellow('--color, -c')}       Behave as FORCE_COLOR set to given value (0, 1, 2, 3, 256, 16m).
```

Comment:
I don't think the color flag supports levels. And it should be made clear that value is optional. And it doesn't behave like `FORCE_COLOR`.

Reply:
You're right. 
I addressed it turning it into a boolean flags, and adding test to ensure the `--color=true` and `--color=always` works as expected.